Snakefile

```diff
@@ -1,15 +1,16 @@
-from config.ScibConfig import *
+from ScibConfig import *
 
 #configfile: "config.yaml"
 cfg = ParsedConfig(config)
 wildcard_constraints:
-    hvg = "hvg|full_feature"
+    hvg="hvg|full_feature",
+    scaling="scaled|unscaled",
+    method="[^/]+"
 
 include: "scripts/preprocessing/Snakefile"
 include: "scripts/integration/Snakefile"
 include: "scripts/metrics/Snakefile"
 include: "scripts/visualization/Snakefile"
-include: "scripts/metrics/cc_score/Snakefile"
```

Comment:
was this something that is just not used anymore? I.e. a separate cc scoring rule you used for testing?

Reply:
We decided to remove cc_score completely, as we didn't end up needing it